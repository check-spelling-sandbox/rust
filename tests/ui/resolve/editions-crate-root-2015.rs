// edition:2015

mod inner {
    fn global_inner(_: ::nonexistent::Foo) {
        //~^ ERROR failed to resolve: maybe a missing crate `nonexistent`?
    }
    fn crate_inner(_: crate::nonexistent::Foo) {
        //~^ ERROR failed to resolve: maybe a missing crate `nonexistent`?
    }

    fn bare_global(_: ::nonexistent) {
        //~^ ERROR cannot find type `nonexistent` in the crate root
    }
    fn bare_crate(_: crate::nonexistent) {
        //~^ ERROR cannot find type `nonexistent` in the crate root
    }
}

fn main() {

}
