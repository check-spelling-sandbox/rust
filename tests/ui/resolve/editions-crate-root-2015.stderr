error[E0433]: failed to resolve: maybe a missing crate `nonexistent`?
  --> $DIR/editions-crate-root-2015.rs:4:26
   |
LL |     fn global_inner(_: ::nonexistent::Foo) {
   |                          ^^^^^^^^^^^ maybe a missing crate `nonexistent`?
   |
   = help: consider adding `extern crate nonexistent` to use the `nonexistent` crate

error[E0433]: failed to resolve: maybe a missing crate `nonexistent`?
  --> $DIR/editions-crate-root-2015.rs:7:30
   |
LL |     fn crate_inner(_: crate::nonexistent::Foo) {
   |                              ^^^^^^^^^^^ maybe a missing crate `nonexistent`?
   |
   = help: consider adding `extern crate nonexistent` to use the `nonexistent` crate

error[E0412]: cannot find type `nonexistent` in the crate root
  --> $DIR/editions-crate-root-2015.rs:11:25
   |
LL |     fn bare_global(_: ::nonexistent) {
   |                         ^^^^^^^^^^^ not found in the crate root

error[E0412]: cannot find type `nonexistent` in the crate root
  --> $DIR/editions-crate-root-2015.rs:14:29
   |
LL |     fn bare_crate(_: crate::nonexistent) {
   |                             ^^^^^^^^^^^ not found in the crate root

error: aborting due to 4 previous errors

Some errors have detailed explanations: E0412, E0433.
For more information about an error, try `rustc --explain E0412`.
